### PR TITLE
Fix issue with choices on JSONArrayFormField

### PR DESCRIPTION
**Pull request by smk4664**
Also apply sort to channel_width and allowed_channel_list during factories.

# Closes #<ISSUE NUMBER GOES HERE>
# What's Changed

# Screenshots

# TODO

- [ ] Explanation of Change(s)
- [ ] Added change log fragment(s) (for more information see [the documentation](https://docs.nautobot.com/projects/core/en/stable/development/#creating-changelog-fragments))
- [ ] Attached Screenshots, Payload Example
- [ ] Unit, Integration Tests
- [ ] Documentation Updates (when adding/changing features)
- [ ] Example App Updates (when adding/changing features)
- [ ] Outline Remaining Work, Constraints from Design

### CODE REVIEW

**Comment by glennmatthews** [comment]
Test failure looks like bad data assumptions uncovered by the change in factory data - I thought we'd fixed that specific one already though -- maybe need to merge latest `next` into the feature branch?

File: nautobot/wireless/factory.py [hunk 1]
```diff
@@ -68,16 +69,19 @@ class Meta:
 class RadioProfileFactory(PrimaryModelFactory):
     class Meta:
         model = models.RadioProfile
-        exclude = ("has_description",)
 
     name = UniqueFaker("word")
     frequency = factory.Faker("random_element", elements=RadioProfileFrequencyChoices.values())
-    channel_width = factory.Faker("random_elements", elements=RadioProfileChannelWidthChoices.values(), unique=True)
+    channel_width = factory.LazyFunction(
+        lambda: sorted(faker.Faker().random_elements(RadioProfileChannelWidthChoices.values(), unique=True))
```
[the remark this answers is not in the document]
Comment (smk4664) [reply nautobot/wireless/factory.py:76]: factory.Faker cannot be sorted. in order to use factory.Faker, I found the only way to sort it was to save it to a temp field and then apply sorted. We use faker.Faker in other factory cases.

**Comment by smk4664** [comment]
> Test failure looks like bad data assumptions uncovered by the change in factory data - I thought we'd fixed that specific one already though -- maybe need to merge latest `next` into the feature branch?

I can try that. I was going to save that for later to reduce the need for me to merge multiple times.

File: nautobot/wireless/factory.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (glennmatthews) [reply nautobot/wireless/factory.py:76]: OK, I must be misremembering something then. Thanks.

**Review by glennmatthews — APPROVED** [review]
gonna assume the merge here is correct (if not, we'll catch it when the feature branch drops back into `next`), passing CI is good enough for me.

**Pull request merged**